QEfficient/utils/generate_inputs.py

```diff
@@ -53,6 +53,7 @@ def prepare_pytorch_inputs(self):
         input_ids = inputs["input_ids"]
         batch_size, input_len = input_ids.shape
         inputs.pop("attention_mask")
+        inputs.pop("token_type_ids", None)

```

Comment:
don't we need to do this in prepare_ort_inputs and also in text_generation script?
Did you test the model that has this issue with infer CLI API?

Reply:
I had tested with export command provided in the jira but 180B model export started but failed due to RAM issue. added the changes for ort and aic runs as well. and tested infer command for single layer model.